plugins/PostProcessingPlugin/plugin.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Post Processing",
     "author": "Ultimaker",
-    "version": "2.2.1",
+    "version": "2.3.0",
```

Comment:
Heh, this version was probably still 2.2 to match Cura version 2.2. The version numbers of built-in plug-ins don't need to change (and if they change it would also need to change in `resources/bundled_packages/cura.json`). I suggest you revert this version increment.

Reply:
Yes, I've tested on my Cura and it failed to load... I'll revert this...